tests/integration/modifier-managers/oo-modifiers-classic-test.js

```diff
@@ -4,11 +4,24 @@ import { render, settled } from '@ember/test-helpers';
 import Service, { inject as service } from '@ember/service';
 import hbs from 'htmlbars-inline-precompile';
 import { Modifier } from 'ember-oo-modifiers';
+import { registerDeprecationHandler } from '@ember/debug';
+
+let isRegistered = false;
+let deprecations;
 
 module('Integration | Modifier Manager | oo modifier (classic)', function(hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function() {
+    deprecations = [];
+    if (!isRegistered) {
+      registerDeprecationHandler((message, options, next) => {
```

Comment:
Wow, Ember gives you a register function but not an unregister function?!

Since this is essentially a global state in a test how do you feel about moving the registration to a test helper? Maybe something like this:

```js
setupDeprecartionTest(hooks);
```

Maybe look like:

```js
// tests/helpers/deprecations.js
import { registerDeprecationHandler } from '@ember/debug';

let deprecations;

registerDeprecationHandler((message, options, next) => {
  if (deprecations) {
    deprecations.push(message);
  } else {
    next(message, options);
  }
});

export default function(hooks) {
  hooks.beforeEach(function() {
    deprecations = [];
    this.deprecations = deprecations;
  });

  hooks.afterEach(function() {
    deprecations = null;
  });
}
```